sdks/typescript/src/apache_beam/runners/portable_runner/runner.ts

```diff
@@ -137,6 +148,11 @@ export class PortableRunner extends Runner {
     return await call.response;
   }
 
+  async cancelJob(jobId: string) {
```

Comment:
ditto: "does it make sense to pass a completion callback to the cancel function itself?"

Reply:
Same.